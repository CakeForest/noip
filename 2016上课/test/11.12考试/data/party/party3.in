8 10
2 1
3 2
4 3
5 2
6 5
7 6
8 6
1 2 4
2 1 10
3 1 1
4 1 7
5 1 10
6 2 3
7 2 10
8 2 1
5 2 9
4 1 5
10
2
1
2
1
3
5
1
5
6
2
