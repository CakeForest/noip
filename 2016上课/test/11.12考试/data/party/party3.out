10
0
10
0
11
9
0
9
12
10
